utils/src/shell_wrapper.rs

```diff
@@ -58,5 +99,12 @@ fn main() {
     let result = real_main();
 
     env::set_var("RUST_BACKTRACE", "1");
-    eprintln!("[shell-wrapper] Error: {:?}", result.unwrap_err());
+    let err = result.unwrap_err();
+
+    eprintln!("[shell-wrapper] Error: {:?}", &err);
+
+    // Write the result to /tmp/shell-wrapper_crash_<pid>.log
+    let pid = std::process::id();
+    let output_file = format!("/tmp/shell-wrapper_crash_{}.log", pid);
```

Comment:
Can we log to the journal instead?

Reply:
It was a quick and dirty implementation so I could see what was going on. Journal sounds like a way better idea. I've changed it to do that instead